Added audit logs, runtime logs, and updated job management (#1888)
vesoft-inc/nebula-docs: docs-2.0/nebula-dashboard-ent/4.cluster-operator/cluster-information/audit-log.md

# Audit log

> The NebulaGraph audit logs store all operations received by graph service in categories. Dashboard Enterprise Edition allows you to quickly view audit logs.

Reviewer: ```suggestion
The NebulaGraph audit logs store and categorize all operations performed on the Graph service. Dashboard Enterprise Edition allows you to quickly view audit logs.
```
